packages/wdio-cli/src/utils.ts

```diff
@@ -376,11 +372,11 @@ export async function detectCompiler(answers: Questionnair) {
 
 /**
  * Check if package is installed
- * @param {string} package to check existance for
+ * @param {string} package to check existence for
  */
-export async function hasPackage(pkg: string) {
+export function hasPackage(pkg: string) {
     try {
-        await resolve(pkg, import.meta.url)
+        require.resolve(pkg)
```

Comment:
Why this change? `require` doesn't work in ESM

Reply:
Because `import.meta.url` does not work in CJS.